except_star.md

```diff
@@ -857,8 +857,24 @@ to be updated.
 
 ## Reference Implementation
 
-[An experimental implementation](https://github.com/iritkatriel/cpython/tree/exceptionGroup-stage5).
-
+We developed these concepts (and the examples for this PEP) with
+[an experimental implementation](https://github.com/iritkatriel/cpython/tree/exceptionGroup-stage5).
```

Comment:
an experimental -> the reference ?

Reply:
I don't know what "reference" means here. It sounds like it might mean "this is what we want to merge", which would be too much.

Reply:
"Reference implementation" means that it works and people can try it out and poke and prod it and read the source code to answer questions about specifics that are hard to derive from reading the spec.

It also means "we have implemented this successfully so we know there are no hidden difficulties in the implementation that aren't apparent from the spec".

It does not have to mean "ready to merge". So I think you're using it correctly.